docs/source/external-datasets/index.rst

```diff
@@ -0,0 +1,13 @@
+=================
+External Datasets
+=================
+
+.. toctree::
+   :hidden:
```

Comment:
🔵 Would exposing this table of contents be helpful, or do you figure it's redundant with the sidebar?

My initial reaction on pulling up the docs was "oh, that's it?", and then I noticed the arrow on the bottom right pointing that took me to "Managing External Datasets" - and only afterwards did I realize I could have used the sidebar to find the child pages.

I'm sure we'll iron out our "docs best practices" when we kick off the next sprint, so I'm okay with leaving this as is for now.

Reply:
I agree in this case it would be helpful. But in longer sections it can be a gigantic wall of text because it includes all subheadings. Maybe a collapsible like this would do it:


<details>
<summary>hey click me</summary>
<ul>
<li>this is my</li>
<li>favorite trick on github</li>
</ul>
</details>

Reply:
We can try replacing `:hidden:` with `:maxdepth: 2` or `:maxdepth: 1` and see how that looks:
<details>
<summary><code>:maxdepth: 1</code></summary>

![Screen Shot 2022-10-28 at 5 06 36 PM](https://user-images.githubusercontent.com/1189602/198752407-32cfd797-e96e-4d5c-ae60-fbbe909eb4fa.png)
</details>
<details>
<summary><code>:maxdepth: 2</code></summary>

![Screen Shot 2022-10-28 at 5 06 57 PM](https://user-images.githubusercontent.com/1189602/198752420-202a38ff-89b9-498b-8442-6d6b46f945e7.png)
</details>

(I like your trick 💯)

Reply:
I missed most of the discussion yesterday. Did we talk about anything relevant to this?